include/openthread/tcp.h

```diff
@@ -0,0 +1,642 @@
+/*
+ *  Copyright (c) 2021, The OpenThread Authors.
+ *  All rights reserved.
+ *
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions are met:
+ *  1. Redistributions of source code must retain the above copyright
+ *     notice, this list of conditions and the following disclaimer.
+ *  2. Redistributions in binary form must reproduce the above copyright
+ *     notice, this list of conditions and the following disclaimer in the
+ *     documentation and/or other materials provided with the distribution.
+ *  3. Neither the name of the copyright holder nor the
+ *     names of its contributors may be used to endorse or promote products
+ *     derived from this software without specific prior written permission.
+ *
+ *  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ *  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ *  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ *  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+ *  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ *  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *  POSSIBILITY OF SUCH DAMAGE.
+ */
+
+/**
+ * @file
+ * @brief
+ *  This file defines the OpenThread TCP API.
+ *
+ */
+
+#ifndef OPENTHREAD_TCP_H_
+#define OPENTHREAD_TCP_H_
+
+#include <openthread/ip6.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @addtogroup api-tcp
+ *
+ * @brief
+ *   This module includes functions that control TCP communication.
+ *
+ * @{
+ *
+ */
+
+/**
+ * A linked buffer structure for use with TCP.
+ *
+ * A single otLinkedBuffer structure references an array of bytes in memory,
+ * via mData and mLength. The mNext field is used to form a chain of
+ * otLinkedBuffer structures.
+ *
+ */
+typedef struct otLinkedBuffer
+{
+    struct otLinkedBuffer *mNext;  ///< Pointer to the next linked buffer in this chain, or NULL if this is the end of the chain
+    const uint8_t *mData;          ///< Pointer to data referenced by this linked buffer
+    uint16_t mLength;              ///< Length of this linked buffer
+} otLinkedBuffer;
+
+struct otTcpEndpoint;
+typedef struct otTcpEndpoint otTcpEndpoint;
+
+/**
+ * @brief This callback informs the application that the TCP 3-way handshake is
+ * complete and that the connection is now established.
```

Comment:
Sorry for not being clear. You don't need to include the `@brief`. You just need to have an empty line after the first sentence.
```suggestion
 * This callback informs the application that the TCP 3-way handshake is
 * complete and that the connection is now established.
```

Reply:
OK. I just pushed a commit fixes this throughout.